docs/release_notes.rst

```diff
@@ -1,6 +1,36 @@
 Percona Toolkit
 ***************
 
+v3.5.1 released 2023-01-
+==============================
+
+New Features
+------------------------------------------------------------
+
+* :jirabug:`PT-2134`: Adds support for PostgreSQL and MySQL operators in pt-k8s-debug-collector
```

Comment:
```suggestion
* :jirabug:`PT-2134`: Adds support for PostgreSQL and MySQL operators in pt-k8s-debug-collector

- **Incompatible change:** Option `--resource` now supports options `pxc`, `ps`, `psmdb`, `psql`, and `none`. Default is `none`.
- **Incompatible change:** Output file `pt-summary.txt` was renamed to `summary.txt` to avoid confusion with the pt-summary tool.
- Two new options have been added: `--kubeconfig` and `--forwardport.` These allow specifying custom configuration files for kubectl and custom port when collecting summary from the database.
- Test case now uses` --kubeconfig`, `--forwardport` options, and environment variables `KUBECONFIG_PXC`, `KUBECONFIG_PS`, `KUBECONFIG_PSMDB`, `KUBECONFIG_PSQL`, `FORWARDPORT` which allow more flexibility.
- Summaries are now collected for all Percona operators unless `--resources=none` is specified. For PXC, MySQL, and MongoDB, the corresponding summary tools (pt-mysql-summary and pt-mongodb-summary) must be in PATH.
- README.md was replaced with a link to README.rst to avoid duplicate effort while working on the tool's documentation.

```

Reply:
Updated

Reply:
Updated